src/test/java/org/beanpod/switchboard/controller/StreamLogControllerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.beanpod.switchboard.controller;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import org.beanpod.switchboard.dao.StreamLogDaoImpl;
+import org.beanpod.switchboard.fixture.StreamLogFixture;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.openapitools.model.StreamLogModel;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class StreamLogControllerTest {
```

Comment:
I agree that this could be part of `LogController`. Or would that be too many responsibilities in one class? It would make sense though if we change the endpoint to `/log/stream/{id}`.